spec/rails_helper.rb

```diff
@@ -1,14 +1,20 @@
 require "spec_helper"
+
 ENV["RAILS_ENV"] ||= "test"
 require File.expand_path("../../config/environment", __FILE__)
 
-abort("The Rails environment is running in production mode!") if Rails.env.production?
-require "rspec/rails"
-
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
 ActiveRecord::Migration.maintain_test_schema!
 
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
We don't add new things to this file. We add new settings in `spec/support/*`

Also, you have shoulda-matchers set up already at: https://github.com/hooroo/domus/blob/master/spec/support/shoulda_matchers.rb

Reply:
👍